docs/standards/FAQ/Channel-Nonce.md

```diff
@@ -0,0 +1,60 @@
+---
+title: What are multi-channel nonces
+sidebar_position: 2
+---
+
+
+This concept was taken from [Out of order execution](https://github.com/amxx/permit#out-of-order-execution).
+
+Using nonces prevent old signed transactions from being replayed again (replay attacks). A nonce is an arbitrary number that can be used just once in a transaction.
+
+#### Problem of Sequential Nonces
+
+With native transactions, nonces are strictly sequential. This means that messages with sequential nonces must be executed in order. For instance, in order for message number 4 to be executed, it must wait for message number 3 to complete.
+
+However, **sequential nonces come with the following limitation**:
+
+Some users may want to sign multiple message, allowing the transfer of different assets to different recipients. In that case, the recipient want to be able to use / transfer their assets whenever they want, and will certainly not want to wait on anyone before signing another transaction.
+
+ This is where **out-of-order execution** comes in.
+
+#### Introducing multi-channel nonces
+
+Out-of-order execution is achieved by using multiple independent channels. Each channel's nonce behaves as expected, but different channels are independent. This means that messages 2, 3, and 4 of `channel 0` must be executed sequentially, but message 3 of channel 1 is independent, and only depends on message 2 of `channel 1`.
+
+The benefit is that the signer key can determine for which channel to sign the nonces. Relay services will have to understand the channel the signer choose and execute the transactions of each channel in the right order, to prevent failing transactions.
+
+#### Nonces in the KeyManager
+
+The Key Manager allows out-of-order execution of messages by using nonces through multiple channels.
+
+ Nonces are represented as `uint256` from the concatenation of two `uint128` : the `channelId` and the `nonceId`.
+
+ - left most 128 bits : `channelId`
+ - right most 128 bits: `nonceId`
+
+![multi-channel-nonce](https://user-images.githubusercontent.com/31145285/133292580-42817340-104e-48c5-832b-533842b98d26.jpg)
+
+                        Example of multi channel nonce, where channelId = 5 and nonceId = 1 
+
+
```

Comment:
To center text do this instead:
![image](https://user-images.githubusercontent.com/477945/141321022-9c1f5cf4-123a-41f0-9ecb-b6a34d7123b5.png)


```suggestion
![multi-channel-nonce](https://user-images.githubusercontent.com/31145285/133292580-42817340-104e-48c5-832b-533842b98d26.jpg)

<p align="center">
<i>Example of multi channel nonce, where channelId = 5 and nonceId = 1</i>
</p>
```

Otherwise it looks like code block:

![image](https://user-images.githubusercontent.com/477945/141320685-b3cd6824-812f-4b0c-8afc-d67192a71c08.png)

Reply:
It may make sense to put this image in the docs repo instead of loading it from GitHub?